### DISCUSSION TITLE
Particles emission not random when instanced from same scene?

### DISCUSSION THREAD

**Discussion by elvisish** [godotengine/godot-proposals#4094] [3D]
I've been trying to instance a couple of particles from the same saved scene and change the colour of the second instance, but they override the visibility of the first instance as the particles are emitted in the exact same way. Is there a way of randomising the seed of emission for each instance?

**Comment by Zireael07** [comment]
Can't you save particle parameters as .tres and load from that? I have smoke/snow particles that only differ in some parameters, including albedo, and it works.

**Comment by azur-wolve** [comment]
Maybe "make unique" could solve the problem.

**Comment by eon-s** [comment]
For any resource, if you are instancing scenes, the Resources are shared by reference, so if you want new instances you need to mark these (particles) Resources as `Local to scene` (in the resource part in the inspector).

As others mentioned before, there is a dropdown option in the resource that let you `make unique` the resource and is useful to create duplicates of a resource if you "copypasted" nodes.

Via code you may need to `duplicate` the instance.
